### PR TITLE
CourseRun slug and filtering fixes

### PR DESCRIPTION
**Pull request by clintonb**

### CODE REVIEW

File: course_discovery/apps/course_metadata/data_loaders/marketing_site.py [hunk 1]
```diff
@@ -400,18 +400,12 @@ def create_course_run(self, course, data):
         }
 
         try:
-            course_run, created = CourseRun.objects.update_or_create(key__iexact=key, defaults=defaults)
+            course_run, __ = CourseRun.objects.update_or_create(key__iexact=key, defaults=defaults)
```
Comment (mikedikan) [inline course_discovery/apps/course_metadata/data_loaders/marketing_site.py:403]: why is this changed to __ ?

Comment (clintonb) [reply course_discovery/apps/course_metadata/data_loaders/marketing_site.py:403]: That is the Pythonic way of saying the value is not used.

**Comment by mikedikan** [comment]
👍

**Comment by bderusha** [comment]
belated 👍
